### PR TITLE
JIP-3 Implement Actions Bar

### PR DESCRIPTION
**Pull request by jpfranci**
# Changes
* Implemented actions bar with autocomplete that allows sorting, filtering by tags, and creation of posts
* The filtering by tags is done by first providing a memoized default list of tags and more searches can reveal more options (like google maps)
* The search input is debounced, so we don't overwhelm our server lmao

### Actions Bar
image

### Default Options
image

### CODE REVIEW

File: packages/client/mock-server/db-sample.json [hunk 1]
```diff
@@ -16,5 +16,28 @@
...
+      "name": "lucky is better"
+    },
+    {
+      "name": "cats"
+    },
+    {
+      "name": "pc parts"
+    },
+    {
+      "name": "clean mustaches"
```
Comment (dryu99) [inline packages/client/mock-server/db-sample.json:40]: exqueeze me

**Comment by srhsrhsrhsrh** [comment]
i can't approve of your past searches

**Review by dryu99 — APPROVED** [review]
lgtm 👻

**Review by srhsrhsrhsrh — APPROVED** [review]
LGTM LEMME RESOLVE MERGE CONFLICTS AAAAAA

**Pull request merged**